packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.base.tsx

```diff
@@ -213,7 +235,9 @@ export class SearchBoxBase extends BaseComponent<ISearchBoxProps, ISearchBoxStat
   };
 
   private _onBlur = (ev: React.FocusEvent<HTMLInputElement>): void => {
-    this._events.off(this._rootElement.current, 'blur');
+    if (this._events) {
+      this._events.off();
```

Comment:
Since you are re-creating this in `_onFocusCapture` you might prefer to `dispose()` tho this works and won't leak as far as I can tell.